src/main/java/net/masterthought/cucumber/generators/FeaturesOverviewPage.java

```diff
@@ -20,6 +21,8 @@ public String getWebPage() {
 
     @Override
     public void prepareReport() {
+        // Check if chart is to be rendered
```

Comment:
Remove this comment as:
- it is not needed
- it is not true comparing to code :)

Reply:
Comment removed.